Mathlib/MeasureTheory/Integral/Pi.lean

```diff
@@ -3,24 +3,35 @@ Copyright (c) 2023 Xavier Roblot. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Xavier Roblot
 -/
+import Mathlib.Analysis.SpecialFunctions.PolarCoord
 import Mathlib.MeasureTheory.Constructions.Pi
 import Mathlib.MeasureTheory.Integral.Prod
 
 /-!
 # Integration with respect to a finite product of measures
 
+## Fubini theorem
+
 On a finite product of measure spaces, we show that a product of integrable functions each
 depending on a single coordinate is integrable, in `MeasureTheory.integrable_fintype_prod`, and
 that its integral is the product of the individual integrals,
 in `MeasureTheory.integral_fintype_prod_eq_prod`.
+
+## Polar coordinates change of variables
+
+The polar coordinates change of variables formula for the Lebesgue integral for a function
+defined on the product space `ι → ℂ`, see `integral_comp_pi_polarCoord_symm`.
+
```

Comment:
I'm not sure this really belongs to this file (it is much more specialized than the generic material already there). What about creating a new file for this?

Reply:
Ok. So I have created a directory `Analysis.SpecialFunctions.PolarCoord`, move the file `Analysis.SpecialFunctions.PolarCoord.lean` to `Analysis.SpecialFunctions.PolarCoord.Basic` and put the new results in the file `Analysis.SpecialFunctions.PolarCoord.Pi`. 

I also added the proof of the result for `f : (ι → ℝ × ℝ) → ℝ≥0∞` and deduce the complex case from it.

Reply:
I hope it's better that way. 
Thanks for the review!